src/Temporalio/Common/Constants.cs

```diff
@@ -0,0 +1,13 @@
+namespace Temporalio.Common
+{
+    /// <summary>
+    /// Constants needed by the Temporal SDK.
+    /// </summary>
+    internal static class Constants
+    {
+        /// <summary>
+        /// Prefix for reserved handler and definition names.
+        /// </summary>
+        internal const string ReservedNamePrefix = "__temporal";
```

Comment:
If you can think of any other class, would be ok moving there (e.g. even `TemporalRuntime` since it's internal) instead of a whole new class/file for a single constant.

Reply:
Well there's always going to be something that'd be the first reason to have a common place for constants - no where else felt like an accurate fit, but I can move it to Runtime.